fontc/src/args.rs

```diff
@@ -31,6 +31,12 @@ pub struct Args {
     #[arg(long, default_value = "false")]
     pub flatten_components: bool,
 
+    /// Whether to write a visualization of threadpool execution of tasks.
+    ///
+    /// See <https://github.com/googlefonts/fontc/pull/443>
+    #[arg(long, default_value = "false")]
+    pub emit_threads: bool,
```

Comment:
maybe `emit_timing` would be a clearer name for this option? Of course the timing is per-thread since we run tasks in parallel..

Reply:
Naming is the hardest part :D I like your naming better, will update.